logger/log.go

```diff
@@ -73,16 +90,19 @@ func CreateLogger(level zapcore.Level, encoding string) *zap.Logger {
 		EncodeCaller:   zapcore.ShortCallerEncoder,
 	}
 	conf := zap.Config{
-		Level:            zap.NewAtomicLevelAt(level),
-		Development:      false,
-		Sampling:         nil,
-		Encoding:         encoding,
-		EncoderConfig:    encoderConf,
-		OutputPaths:      []string{"stdout"},
-		ErrorOutputPaths: []string{"stdout"},
+		Level:       zap.NewAtomicLevelAt(level),
+		Development: false,
+		Sampling: &zap.SamplingConfig{
+			Initial:    150,
```

Comment:
I added this, since I thought it's prudent to ignore more than 150 duplicate logs in the same second. If it happens - it means something is wrong